### DISCUSSION TITLE
RabbitMQ Backup and Restore

### DISCUSSION THREAD

**Discussion by 3rk1n** [rabbitmq/cluster-operator#1629] [Q&A]
Hi,

We have an AKS ecosystem and we are using RabbitMQ as a cluster operator (https://github.com/rabbitmq/cluster-operator)
RabbitMQ has an own namespace and we want to backup all RabbitmQ regularly to restore any time.

We succeeded to backup RabbitMQ via Azure policies but we can not restore the queue.

Is there any way to back and restore all RabbitMQ?

Best wishes.

**Comment by lukebakken** [comment]
> Is there any way to back and restore all RabbitMQ?

My advice is - don't. Export your definitions and have the ability to re-build your cluster from scratch.
